docs/api_reference/src/quantities.tex

```diff
@@ -109,19 +264,56 @@
 
 export namespace mp_units {
 
+// \ref{qty.chrono}, \tcode{std::chrono} compatibility
+
+template<typename Rep, typename Period>
+struct quantity_like_traits<std::chrono::duration<Rep, Period>>;
+
+template<typename Clock>
+struct chrono_point_origin_;
+template<typename Clock>
+constexpr chrono_point_origin_<Clock> @\libglobal{chrono_point_origin}@{};
+
+template<typename Clock, typename Rep, typename Period>
+struct quantity_point_like_traits<
+  std::chrono::time_point<Clock, std::chrono::duration<Rep, Period>>>;
+
 }
 \end{codeblock}
 
 \rSec1[qty.helpers]{Helpers}
 
+\rSec2[qty.helpers.non.types]{Non-types}
+
+\begin{itemdecl}
+template<typename T>
+concept @\defexposconceptnc{tag-type}@ = type_is_empty(^T) && type_is_final(^T);  // \expos
+\end{itemdecl}
+
+\begin{itemdecl}
+consteval bool @\exposidnc{is-specialization-of}@(  // \expos
+  std::meta::info type, std::meta::info template_name);
+\end{itemdecl}
```

Comment:
This is definitely an overuse of reflection.
Unless you can directly ask the compiler for the answer,
it should be more efficient to use TMP.
But then again, this is an implementation-defined entity,
and implementors should already have such a facility for other standard things.

Reply:
As I wrote elsewhere, everything that looks new or odd to LEWG will be a problem during standardization. People often fear new things, especially when they do not understand them yet. Our library is already considered too complex; let's not add another level of complexity if we really do not have to.

Reply:
Ah, actually, this might be the most legit use of reflection.

Without reflection or that proposal you linked to,
we need a different trait for each combination of
type/non-type/template template parameter (pack)
that a target template has.
Just like mp-units does.

Having this single _`is-specialization-of`_ greatly simplifies the specification.

Reply:
> Unless you can directly ask the compiler for the answer,

I was thinking of _`is-convertible-to-base-subobject-of`_.
_`is-specialization-of`_ is already as simple and efficient as can be.

Reply:
I will come back to you when I get some feedback about this from LWG.